Templates/Files/Local Files/JSON to Zoho CRM - Customers/long-description.md

```diff
@@ -0,0 +1,65 @@
+This template reads a JSON file containing customer data from a local fileshare and either inserts or updates each customer in Zoho CRM as contacts.
+
+![Template](assets/JSON_to_Zoho_CRM_-_Customers.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Frends agent has access to the local fileshare containing the JSON file.
+- Zoho CRM client ID and client secret should be accessible and provided in the process variables to create the refresh token.
+- Zoho CRM refresh token has been generated and provided in the process variables. A refresh token can be generated with the corresponding template.
+- Zoho CRM regional instances for account and API are known, and the domains are adjusted accordingly.
```

Comment:
I think that applies to every template we've made that uses HTTP Requests - we have to know the URL, so in itself it's kind of self-explanatory. I even thought if we needed to mention that at all! But because I stumbled on that URL change twice, I think it's worth to mention.
Since the URL can vary so much in Zoho CRM cases based on the region, I worded it out this way, but of course it can be changed to be more clear 😄 

"Zoho CRM regional URLs for accounts and API are adjusted accordingly in the process variables"?

Reply:
Hmm, you're probably right. You can leave it as it is, or change it to the one you suggested. 😄